### PR TITLE
init model

### PR DESCRIPTION
**Pull request by VuongLong**

### CODE REVIEW

**Comment by hienbx94** [comment]
OK

**Comment by trantube94** [comment]
Ok

**Comment by hellovietnam93** [comment]
OK

File: Gemfile [hunk 1]
```diff
@@ -1,45 +1,35 @@
...
-# gem 'bcrypt', '~> 3.1.7'
-
-# Use Unicorn as the app server
-# gem 'unicorn'
-
-# Use Capistrano for deployment
-# gem 'capistrano-rails', group: :development
+source "https://rubygems.org"
+
+gem "rails", "4.2.1"
```
Comment (dieunb) [inline Gemfile]: dùng rails 4.2.3

File: Gemfile [hunk 2]
```diff
@@ -1,45 +1,35 @@
...
+gem "uglifier", ">= 1.3.0"
+gem "coffee-rails", "~> 4.1.0"
+gem "bootstrap-sass",       "3.2.0.0"
+gem "will_paginate",           "3.0.7"
+gem "bootstrap-will_paginate", "0.0.10"
+gem "jquery-rails"
+gem "turbolinks"
+gem "jbuilder", "~> 2.0"
+gem "sdoc", "~> 0.4.0", group: :doc
+gem "bcrypt", "~> 3.1.7"
```
Comment (dieunb) [inline Gemfile]: no need, it is included in devise gem

File: Gemfile [hunk 3]
```diff
@@ -1,45 +1,35 @@
...
-# Use Capistrano for deployment
-# gem 'capistrano-rails', group: :development
+source "https://rubygems.org"
+
+gem "rails", "4.2.1"
+gem "sass-rails", "~> 5.0"
+gem "uglifier", ">= 1.3.0"
+gem "coffee-rails", "~> 4.1.0"
+gem "bootstrap-sass",       "3.2.0.0"
+gem "will_paginate",           "3.0.7"
```
Comment (dieunb) [inline Gemfile]: use kaminari gem instead it

File: Gemfile [hunk 4]
```diff
@@ -1,45 +1,35 @@
...
+gem "bootstrap-will_paginate", "0.0.10"
+gem "jquery-rails"
+gem "turbolinks"
+gem "jbuilder", "~> 2.0"
+gem "sdoc", "~> 0.4.0", group: :doc
+gem "bcrypt", "~> 3.1.7"
+gem "rails_config"
+gem "faker",                "1.4.2"
```
Comment (dieunb) [inline Gemfile]: move to development group

File: Gemfile [hunk 5]
```diff
@@ -1,45 +1,35 @@
...
+gem "coffee-rails", "~> 4.1.0"
+gem "bootstrap-sass",       "3.2.0.0"
+gem "will_paginate",           "3.0.7"
+gem "bootstrap-will_paginate", "0.0.10"
+gem "jquery-rails"
+gem "turbolinks"
+gem "jbuilder", "~> 2.0"
+gem "sdoc", "~> 0.4.0", group: :doc
+gem "bcrypt", "~> 3.1.7"
+gem "rails_config"
```
Comment (dieunb) [inline Gemfile]: use logic_setting gem instead of it

File: Gemfile [hunk 6]
```diff
@@ -1,45 +1,35 @@
...
 group :development, :test do
-  # Call 'byebug' anywhere in the code to stop execution and get a debugger console
-  gem 'byebug'
-
-  # Access an IRB console on exception pages or by using <%= console %> in views
-  gem 'web-console', '~> 2.0'
-
-  # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
-  gem 'spring'
+  gem "byebug", "3.4.0"
```
Comment (dieunb) [inline Gemfile]: use byebug instead of it

File: Gemfile [hunk 7]
```diff
@@ -1,45 +1,35 @@
...
+  gem "mysql2", "~> 0.3.11"
```
Comment (dieunb) [inline Gemfile]: move outside this block, we will use it for production evn, and you no need define version. Always use newest version

File: Gemfile [hunk 8]
```diff
@@ -1,45 +1,35 @@
...
+gem "jquery-rails"
+gem "turbolinks"
+gem "jbuilder", "~> 2.0"
+gem "sdoc", "~> 0.4.0", group: :doc
+gem "bcrypt", "~> 3.1.7"
+gem "rails_config"
+gem "faker",                "1.4.2"
+gem "carrierwave",             "0.10.0"
+gem "mini_magick",             "3.8.0"
+gem "fog",                     "1.32.0"
```
Comment (dieunb) [inline Gemfile]: no need

**Comment by minhld1709** [comment]
OK

File: .gitignore [hunk 9]
```diff
@@ -11,11 +11,13 @@ capybara-*.html
 **.orig
 rerun.txt
 pickle-email-*.html
+/test
+/spec
```
Comment (hellovietnam93) [inline .gitignore]: remove this line

**Pull request merged**